.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Description of changes  
+Provide a description of what this PR does. What bug does it fix, or what feature does it add? Do you expect that this PR will change answers, and if so, under what circumstances?
```

Comment:
If this is a physics innovation which addresses a forecast issue, It will be great if the contributor adds in a reference.

Reply:
Good idea. I updated the template to encourage developers to include references, if applicable.